Source/WebCore/platform/network/OrganizationStorageAccessPromptQuirk.h

```diff
@@ -61,7 +61,7 @@ inline void add(Hasher& hasher, const OrganizationStorageAccessPromptQuirk& quir
 
 struct OrganizationStorageAccessPromptQuirkHashTraits : SimpleClassHashTraits<OrganizationStorageAccessPromptQuirk> {
     static const bool hasIsEmptyValueFunction = true;
-    static const bool emptyValueIsZero = true;
+    static const bool emptyValueIsZero = false;
```

Comment:
This change isn't described in the log.

Reply:
That was the drive-by comment, but I'll fix that in #22340 instead.